### ISSUE TITLE
Tee Off

### ISSUE THREAD

**Issue by wicksand420** [Rinnegatamante/flycast-vita-Compatibility#183]
# Game summary
- Game name: Tee Off
- Game version: cdi usa

# flycast summary
- Version: v.0.1

# Issues

Black background with random textures appearing, not playable
# Screenshots
![](https://?)

# Recommended labels

- Ingame (Low)

**Comment by Rinnegatamante** [comment]
SMC Accuracy and SMC Performance?

**Comment by wicksand420** [comment]
Yeah, with all 3 cores, same thing, this is the worse rendered game I've seen so far, and I've been through a lot of these so far
